wagtail_localize/tests/test_submit_translations.py

```diff
@@ -579,6 +579,30 @@ def test_post_submit_page_translation_from_page_with_privacy_settings(self):
             translated_page.view_restrictions.first().pk, view_restriction.pk
         )
 
+    def test_post_submit_page_translation_from_draft_source_still_draft(self):
+        custom_page = make_test_page(
+            self.en_homepage,
+            cls=TestWithTranslationModeDisabledPage,
```

Comment:
Any particular reason for picking `TestWithTranslationModeDisabledPage` here? It isn't _wrong_, but this test isn't specifically about what happens when translation mode is off so it kind of distracts from the intent of the test IMO.